toqito/matrix_props/sk_norm.py

```diff
@@ -227,7 +227,7 @@ def sk_operator_norm(
                     return op_norm * lower_bound, op_norm * upper_bound
 
     # Start the semidefinite programming approach for getting upper bounds.
-    if effort >= 1 and (
+    if effort >= 1 and (  # pylint: disable=too-many-boolean-expressions
```

Comment:
These lines could be refactored. 

https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/too-many-boolean-expressions.html